docs/sources/alerting/fundamentals/notifications/contact-points.md

```diff
@@ -1,9 +1,10 @@
 ---
 aliases:
-  - /docs/grafana/latest/alerting/contact-points/
-  - /docs/grafana/latest/alerting/unified-alerting/contact-points/
-  - /docs/grafana/latest/alerting/fundamentals/contact-points/contact-point-types/
-canonical: https://grafana.com/docs/grafana/latest/alerting/fundamentals/contact-points/
+  - ../../fundamentals/contact-points/
+  - ../../fundamentals/contact-points/contact-point-types/
+  - ../../contact-points/
+  - ../../unified-alerting/contact-points/
```

Comment:
Please add comments with absolute URL for the future.

Reply:
fixed by https://github.com/grafana/grafana/pull/84248/commits/1133f2d73d48d096fedd2b7f76eec40923e86344